gcalbot/gcalbot/handler.go

```diff
@@ -63,8 +63,17 @@ func (h *Handler) HandleCommand(msg chat1.MsgSummary) error {
 	}
 
 	tokens, err := shellquote.Split(cmd)
-	if err != nil {
-		// TODO(marcel): send better error message to user
+	switch err {
```

Comment:
this might be nice in a base class wherever we use shellquote in bots, what do you think?

Reply:
Sounds good, I can do that in this PR